mutation/package.json

```diff
@@ -3,9 +3,7 @@
   "module": "./dist/index.esm.js",
   "types": "./dist/index.d.ts",
   "exports": {
-    "types": "./dist/index.d.ts",
     "import": "./dist/index.mjs",
-    "module": "./dist/index.esm.js",
```

Comment:
why we need to delete module field?

Reply:
https://webpack.js.org/guides/package-exports/#target-environment-independent-packages

I think this field is designed for `node` only.

It prevents packages from being bundled twice when referencing packages with `require()` and `import` at same time. 

So i think we probably don't need this feature here ?

Reply:
Feel it's still cirtical to change here as it could be bundled on server side in nodejs runtime. Also it's not related to the types generation. Would be good to create a separate PR and plan in a minor release instead of carrying it into this PR

Reply:
👌